### ISSUE TITLE
TypeError with Sphinx 5.0

### ISSUE THREAD

**Issue by lukehsiao** [mansenfranzen/autodoc_pydantic#125]
When using autodoc_pydantic with the just-released Sphinix v5.0.0, we get a TypeError:

```
  File "/home/redacted/.venv/lib/python3.8/site-packages/sphinxcontrib/autodoc_pydantic/directives/autodocumenters.py", line 205, in add_content
    super().add_content(more_content, no_docstring)
TypeError: add_content() takes 2 positional arguments but 3 were given
```

This appears to be changed in sphinx's autodoc extension here: https://github.com/sphinx-doc/sphinx/commit/267954edba498dcc132f98c6a6e7fd029e8f6bbb#diff-e43bdd6f8f37a12d2536e09e57c5e8999cb8de18b9c7ba49126f90576c4328acR2666

It seems that `autodoc_pydantic` will also need to adjust accordingly.

**Comment by mansenfranzen** [comment]
@lukehsiao Thanks for reporting this issue. I can confirm that it occurs due to a changed function signature in sphinx 5.0.

**Comment by mansenfranzen** [comment]
@all-contributors please add @lukehsiao for bug

**Comment by mansenfranzen** [comment]
Bug is fixed and new version of `autodoc_pydantic=1.7.1` is released to support `sphinx=5.0`.

**Comment by lukehsiao** [comment]
Amazing, thank you!
